### DISCUSSION TITLE
How can I optimize the dynamics of the connecting lines between nodes to see the relationship between nodes more clearly？

### DISCUSSION THREAD

**Discussion by water110** [Comfy-Org/ComfyUI#3650] [Ideas]
![动态连接线](https://github.com/comfyanonymous/ComfyUI/assets/29881977/31c73f5e-11a5-4ca9-a591-8284da3d7139)
I hope to develop a plug-in similar to this effect, after selecting a node, the node connection line associated with the current node can have a flowing dynamic effect, similar to the above picture. Now encounter a very headache problem, when the node is a lot of nodes, view the connection line between the nodes is very difficult, I hope that someone can develop this plug-in, in addition, if you can develop a one-click layout of all the nodes, so that the whole interface looks more clear and neat, it would be better!
希望可以开发类似这种效果的插件，选中某个节点后，当前节点关联的节点连接线可以有流水的动效，类似于上图一样。现在遇到一个很头疼的问题，当节点很多时，查看节点之间的连接线非常困难，希望有人可以开发这种插件，另外如果能开发出一键排版所有节点，让整个界面看起来更加清晰整洁，就更好了！

**Comment by bongraster** [comment]
With bigger workflows clicking on a node with a lot of connections will make your screen like a Christmas tree.

**Comment by BrechtCorbeel** [comment]
I like it, but please think of the CPU/GPU usage, my firefox already does 20/50 percent CPU and a number of GB Vram, just for being open, it took me to close firefox to get my card running again on a stuck ksampler, because 1080p animatediff is +-24GB
